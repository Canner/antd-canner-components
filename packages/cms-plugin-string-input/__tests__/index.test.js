import React from 'react';
import Input from '../src';
import renderer from 'react-test-renderer';

test('[input] render success with value', () => {
  const component = renderer.create(
    <Input value="this is a test"/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});