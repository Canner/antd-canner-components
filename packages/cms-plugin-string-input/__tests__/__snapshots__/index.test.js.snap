// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[string/input] render success with value 1`] = `
<div
  id="input"
>
  <input
    className="ant-input"
    disabled={false}
    onChange={[Function]}
    onCompositionEnd={[Function]}
    onCompositionStart={[Function]}
    onKeyDown={[Function]}
    type="text"
    value="this is a test"
  />
</div>
`;
